run 003 - rmt 0, cmp 1, exists 0
================================

> [BACKREST_BIN_PATH]/../bin/pg_backrest --config=[TEST_PATH]/db/pg_backrest.conf --stanza=db archive-get 000000010000000100000001 [TEST_PATH]/db/common/pg_xlog/000000010000000100000001
------------------------------------------------------------------------------------------------------------------------------------
 INFO: getting WAL segment 000000010000000100000001
DEBUG:     ArchiveInfo->new(): archiveClusterPath = [TEST_PATH]/backrest/archive/db
ERROR: [130]: archive.info does not exist but is required to get WAL segments
              HINT: Is archive_command configured in postgresql.conf?
              HINT: Use --no-archive-check to disable archive checks during backup if you have an alternate archiving scheme.
DEBUG:     safe exit called, terminating threads

+ supplemental file: [TEST_PATH]/backrest/archive/db/archive.info
-----------------------------------------------------------------
[backrest]
backrest-checksum="[CHECKSUM]"
backrest-format=4
backrest-version="[VERSION-1]"

[db]
db-id=1
db-system-id=1234567890123456789
db-version="9.3"

[db:history]
1={"db-id":1234567890123456789,"db-version":"9.3"}

> [BACKREST_BIN_PATH]/../bin/pg_backrest --config=[TEST_PATH]/db/pg_backrest.conf --stanza=db archive-get 000000090000000900000009 [TEST_PATH]/db/common/pg_xlog/RECOVERYXLOG
------------------------------------------------------------------------------------------------------------------------------------
 INFO: getting WAL segment 000000090000000900000009
DEBUG:     ArchiveInfo->new(): archiveClusterPath = [TEST_PATH]/backrest/archive/db
DEBUG:     Archive->getCheck=>: archiveId = 9.3-1
DEBUG:     File->list: backup:absolute:[TEST_PATH]/backrest/archive/db/9.3-1/0000000900000009, expression ^000000090000000900000009(-[0-f]+){0,1}(\.gz){0,1}$, sort forward
DEBUG:     File->exists: backup:absolute:[TEST_PATH]/backrest/archive/db/9.3-1/0000000900000009
 INFO: 000000090000000900000009 was not found in the archive repository
DEBUG:     safe exit called, terminating threads
